src/_config.ts

```diff
@@ -27,6 +27,17 @@ export function semanticReleaseGit(assets: string[]): SemanticReleasePlugin {
   ];
 }
 
+export function semanticReleaseOpenApi(
```

Comment:
This is only used once, so it could still stay as part of the openapi preset. If kept here, then I would add JSDocs to the function